packages/ui/components/textarea/test/lion-textarea.test.js

```diff
@@ -98,10 +93,13 @@ describe('<lion-textarea>', () => {
     expect(hightWith1Line < hightWith5TextLines).to.equal(true);
   });
 
-  // TODO: To be fixed in 4096171
-  it.skip(`starts growing when content is bigger than "rows"
+  it(`starts growing when content is bigger than "rows"
     'and stops growing after property "maxRows" is reached`, async () => {
-    const el = await fixture(`<lion-textarea></lion-textarea>`);
+    const el = await fixture(html`
+      <lion-textarea>
+        <textarea slot="input" style="box-sizing: content-box;"></textarea>
```

Comment:
Could we keep the old API so that `textarea slot="input"` does not have to be added by users as a mandatory thing to make this example work?

Reply:
I moved this to this part of styling to the `web-test-runner.config.mjs`